### ISSUE TITLE
There are some questions for testing curve25519.fst.

### ISSUE THREAD

**Issue by jnchzy** [project-everest/hacl-star#135]
There were some problems when I tested the file by fstar.exe.The errors are showed in the following figure. I don't know the reason. I am looking forward for your answer. Thank you very much.
![image](https://user-images.githubusercontent.com/20393374/50533263-3dea2880-0b61-11e9-9415-bc210bdba10f.png)

**Comment by beurdouche** [comment]
Hi, please make sure you are using F* and Kremlin on their `stable` branches and this should work.

**Comment by jnchzy** [comment]
Thank you very much. I have replaced fstar.exe by fstar_0.9.6.0_Windows_x64. Before I used fstar.exe of fstar_0.9.5.0_Windows_x64. But there are still some questions. The errors are showed in the following figure. I don't know the reason. I am looking forward for your answer. Thank you very much. 
![image](https://user-images.githubusercontent.com/20393374/50881657-4e31bb80-141d-11e9-95be-adbbe488c027.png)
